assets/app/game_manager.rb

```diff
@@ -156,6 +156,7 @@ def update_game(game)
     @games += [game] if @games.none? { |g| g['id'] == game['id'] }
     @games.reject! { |g| g['id'] == game['id'] } if game['deleted']
     @games.map! { |g| g['id'] == game['id'] ? game : g }
+    store(:game, game, skip: true) if @game&.[]('id') == game['id']
```

Comment:
@daniel-sousa-me 
What exactly is the purpose of this line / why did you include it?

Reply:
Learning Ruby xP

This makes no sense at all to me. I don't know what happened here, but probably something got lost in the middle of a lot of moving things around. This was one of my first PRs (if not the first) and I tried a bunch of things in a bunch of places before I was able to figure out how to make this work. One of the versions the invite page was the normal home page with the game on top. My guess is that it's a leftover from that.

Are you working on something here? If not, I'll look again at this PR and remove this and other dumb stuff I find. I've been meaning to go back to it to make the invite page auto-refresh, so this is a good excuse to do it.

Reply:
If you're working on something around here, feel free to remove this. I'm 99% sure this does not interact at all with the rest of the PR...

Reply:
At some point this line was `store(:game, game, skip: true) if @game && @game['id'] == game['id']`, but probably rubocop or someone complained about that

Reply:
> One of the versions the invite page was the normal home page with the game on top. My guess is that it's a leftover from that.

That would explain this vestige.
It triggered a problem which cost me quite a headache … When I finally nailed it down to this line I couldn’t figure out its purpose. I’ll just delete it for now. Thanks for your help.

Reply:
I'm so sorry. Those bug hunts really suck.

Reply:
It turns out that does indeed do something. When you're on the invite page, if you click on a button, the server sends back data about that game. This line stores it so that the card is updated.

Reply:
Can you describe the problem it was causing?

Reply:
I've reviewed this and that is actually the intended behaviour. Maybe there's some part of the code calling update_game() at the wrong time.

Reply:
A comment on this line stating the reasons it’s there would’ve helped greatly. (I added one in my branch.)
The problem is @game is only set/wiped on page reloads. Getting rid of reloads brings trouble when interacting with new games, but it’s solved.

Reply:
What did you change? I just realise there's a mistake after all. I should be using `@game_data` instead of `@game`

Reply:
Just push your change (or tell me how the line should read). I’ll deal with the implications on my end.

Reply:
https://github.com/tobymao/18xx/pull/3658

Reply:
I ended up using `store(:game, game, skip: true) if @game_data[:id] == game['id']` which seemed to work fine.